.github/workflows/release-workflow.yml

```diff
@@ -0,0 +1,34 @@
+name: native-metrics CI
+
+on:
+  push:
+    tags:
+      - 'v*' # Push events to matching v*, e.g. v1.0, v20.15.10
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
```

Comment:
I think we want to support Node 14 on this module as well

Reply:
Added v14